# General Configuration

# FT_sensor Configuration
use_ft_sensor: true

# Simulation Configuration
use_simulation: false

# Optitrack Configuration
use_optitrack: true

# Pad Configuration
pad:
  # Pad dimensions
  dimensions:
    x: 0.12
    y: 0.14
    z: 0.004
  # Pad origin position
  origin:
    x: 0.0
    y: 0.0
    z: 0.0

# RViz Configuration
rviz:
  #Offset position of the pad in RViz
  position:
    x: -0.06
    y: 0.147
    z: 0.009

# ATI Gamma FT Sensor Configuration
sensor:
  # Default sensor position relative to origin of the pad (front left)
  position:
    x: 0.06
    y: 0.142 # TRY 0.1616
    z: -0.008
  
  simulation:
    # Noise parameters
    noise_magnitude: 0.01

# Calibration Configuration
calibration:
  # Calibration type
  use_calibration: static
  # Valid calibration types
  types:
    - static
    - dynamic
  # Static calibration parameters
  static:
    num_samples: 100

# Processing Configuration
processing:
  # Minimum force magnitude threshold for contact point estimation
  force_threshold: 0.4
  # Time window for force data processing
  time_window: 0.1
  # Padding for trajectory generation TO CHANGE TO ADAPT BASED ON X MIN MAX Y MIN MAX
  padding:
    x: 0.5
    y: 0.2


 
